test/read.spec.ts

```diff
@@ -0,0 +1,40 @@
+// Import Node.js Dependencies
+import * as fs from "node:fs/promises";
+import path from "node:path";
+import os from "node:os";
+
+// Import Third-party Dependencies
+import { expect } from "chai";
+
+// Import Internal Dependencies
+import { read } from "../src/index";
+import { generateDefaultRC } from "../src/rc";
+
+describe("read .nodesecurerc", () => {
```

Comment:
Maybe could we have the case where the `.nodesecurerc` and returns the config?

Reply:
I did not understand 😂